workers/ohsome_quality_analyst/base/report.py

```diff
@@ -58,6 +60,24 @@ def __init__(
         # Results will be written during the lifecycle of the report object (combine())
         self.result = Result(None, None, None)
 
+    @property
+    def __geo_interface__(self) -> FeatureCollection:
+        """An interface which supports GeoJSON Encoding/Decoding.
+
+        Returns a GeoJSON FeatureCollection object.
+        Features are all indicators.
+        Interface Specification: https://gist.github.com/sgillies/2217756
+        GeoJSON Encoding/Decoding: https://github.com/jazzband/geojson#custom-classes
+        """
+        features = []
+        for indicator in self.indicators:
+            features.append(
+                geojson.loads(
+                    geojson.dumps(indicator, default=datetime_to_isostring_timestamp)
+                )
+            )
+        return FeatureCollection(features=features)
```

Comment:
This is not nice code. Idealy one could simply write:
```python
return FeatureCollection(features=[Feature(indicator) for indicator in self.indicator])
```

But this fails because of datetime objects can not be encoded by `json.dump` without explicit defining the default argument (`default=datetime_to_isostring_timestamp)`)

Reply:
An alternative is:
```
        return FeatureCollection(
            features=[indicator.__geo_interface__ for indicator in self.indicators]
        )
```